packages/server/lib/browsers/electron.js

```diff
@@ -121,6 +121,9 @@ module.exports = {
         y: 'browserY',
         devTools: 'isBrowserDevToolsOpen',
       },
+      webPreferences: {
+        sandbox: true,
```

Comment:
we probably should've been running with `sandbox: true` all along, now it has become necessary in order to match Chrome's error behavior to Electron's